maxim-belkin/homebrew-xorg
# Improve GitHub Actions

'audit':
 - set HOMEBREW_NO_ANALYTICS
 - set HOMEBREW_NO_AUTO_UPDATE

'build_bottle' ~> 'test_pull_requests':
 - split workflow into 4 jobs:
   * audit
   * build bottles
   * upload to Bintray
   * check tag in LinuxbrewTestBot
